src/licenses.ts

```diff
@@ -0,0 +1,39 @@
+import {Change, ChangeSchema} from './schemas'
+
+/**
+ * Loops through a list of changes, filtering and returning the
+ * ones that don't conform to the licenses allow/deny lists.
+ * @param {Change[]} changes The list of changes to filter.
+ * @param { { allow?: string[], deny?: string[]}} licenses An object with `allow`/`deny` keys, each containing a list of licenses.
+ * @returns {Array<Change} The list of denied changes.
+ */
+export function getDeniedLicenseChanges(
+  changes: Array<Change>,
+  licenses: {
+    allow?: Array<string>
+    deny?: Array<string>
+  }
+): Array<Change> {
```

Comment:
I really appreciate programs that correct _all_ of the errors and display them all at once. It's really frustrating to get an error, fix it, run the program again, get another error, and repeat. I'm glad you're doing this 😄